src/Tizen.Multimedia.Vision/MediaVision/QrConfiguration.cs

```diff
@@ -78,5 +82,67 @@ public QrConfiguration(QrMode qrMode, ErrorCorrectionLevel ecc, int version)
         /// </summary>
         /// <since_tizen> 4 </since_tizen>
         public int Version { get; }
+
+        /// <summary>
+        /// Gets or sets the embed image absolute path of the Design QR code.
+        /// </summary>
+        /// <exception cref="ArgumentNullException"><paramref name="value"/> is null.</exception>
+        /// <exception cref="ArgumentException"><paramref name="value"/> is zero-length string.</exception>
+        /// <since_tizen> 11 </since_tizen>
+        public string EmbedImagePath
+        {
+            get
+            {
+                return _embedImagePath;
+            }
+            set
+            {
+                ValidationUtil.ValidateIsNullOrEmpty(value, nameof(value));
+
+                _embedImagePath = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the data shape of the Design QR code.
+        /// </summary>
+        /// <remarks>The default value is <see cref="QrShape.Rectangular"/>.</remarks>
+        /// <remarks>If user get value before setting value, DataShape will return null.</remarks>
+        /// <exception cref="ArgumentException"><paramref name="value"/> is not valid.</exception>
+        /// <since_tizen> 11 </since_tizen>
+        public QrShape DataShape
+        {
+            get
+            {
+                return _dataShape;
+            }
+            set
+            {
+                ValidationUtil.ValidateEnum(typeof(QrShape), value, nameof(value));
+
+                _dataShape = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the finder shape of the Design QR code.
+        /// </summary>
+        /// <remarks>The default value is <see cref="QrShape.Rectangular"/>.</remarks>
+        /// <remarks>If user get value before setting value, FinderShape will return null.</remarks>
+        /// <exception cref="ArgumentException"><paramref name="value"/> is not valid.</exception>
```

Comment:
maybe
```suggestion
        /// <exception cref="ArgumentException"><paramref name="value"/> set is not valid.</exception>
```
if you decide to apply that, then the same for lines 89, 90, 111

Reply:
I don't understand the meaning of `set` in here.